pysb/core.py

```diff
@@ -1643,6 +1647,14 @@ def __getitem__(self, key):
         else:
             return self._map[key]
 
+    def __getattr__(self, name):
+        if name == '_map':
+            raise AttributeError()
+        try:
+            return self[name]
+        except KeyError as e:
+            raise AttributeError(e)
```

Comment:
The standard AttributeError message is of the form `'Foo' object has no attribute 'bar'`. This code just produces a message of the form `'bar'`. Maybe the message should be something like `Model has no component 'bar'`, e.g. 
```python
raise AttributeError("Model has no component '%s'" % name)
```

Reply:
Implemented.